### ISSUE TITLE
Cannot build the app

### ISSUE THREAD

**Issue by MarcAnt01** [robmikh/SimpleRecorder#16]
I get multiple errors. Are there additional steps to do?
![image](https://user-images.githubusercontent.com/43090263/84019565-b4f43500-a981-11ea-94f2-0e9810953d02.png)

**Comment by robmikh** [comment]
It looks like you either have an SDK issue or you need to forcibly restore the nuget dependencies.

**Comment by MarcAnt01** [comment]
Yeah, looks like my problem is due to nuget packages.

**Comment by MarcAnt01** [comment]
@robmikh I am having problems with nuget packages, how can I solve them?
![image](https://user-images.githubusercontent.com/43090263/84142941-f8b86e80-aa55-11ea-836f-6e5c3f0b240e.png)

**Comment by robmikh** [comment]
Have you run `nuget restore`? If it can't find the specific version then try to upgrade the packages to a newer one. Unfortunately whenever I get nuget-related errors like these I just kind of flail around until it works, I don't have specific steps...

**Comment by MarcAnt01** [comment]
No, I have jus tried to update them. I have no experience with nuget packages, can you tell me how to do that?

**Comment by robmikh** [comment]
Go to the root directory of the project in a command prompt window and run `nuget restore`. If you don't have nuget in your PATH, you can find a copy [here](https://www.nuget.org/downloads).

**Comment by MarcAnt01** [comment]
I have installed the recommended version from the link you've sent to me, but still nuget gets displayed as an unknown command. Do I have to add it manually to PATH?

**Comment by robmikh** [comment]
You either have to manually add it to your PATH or you can just put the exe in the folder you're going to run it in.

**Comment by MarcAnt01** [comment]
OK, it worked now, but when I try to build the app I have the same issues with nuget packages
